src/main/kotlin/io/zerobase/smarttracing/GraphDao.kt

```diff
@@ -150,7 +150,7 @@ class GraphDao(private val driver: Driver, val phoneUtil: PhoneNumberUtil) {
                 txn.run(
                         """
                         MATCH (o:Organization { id: '${id.value}' })
-                        SET o.multisite = '${state}'
+                        SET o.multisite = 'true'
```

Comment:
no, this was correct. revert your change.

Reply:
See your previous comment regarding reverting back the fix as we are switching databases